app/views/activities/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= link_to user_categories_path do %>
+  <i class="bi bi-arrow-left btn">Back</i>
+<%end%>
+
+<div>
+  <div class="d-flex align">
+    <h3>Total transactions: $<%= @activities.sum { |a| a.amount } %></h3>
+  </div>
+
+  <% if @activities.blank? %>
+    <h3 class="empty">Your Transactions list is currently empty!</h3>
+  <% else %>
+    <ul id="list" class="cards">
+      <% @activities.most_recent.each do |a| %>
+        <li class="card d-flex">
+          <div class="card-body d-flex">
+            <div>
+              <% if @category.icon.attached? %>
+                <%= image_tag url_for(@category.icon), class: "image" %>
+              <% end %>
+
+            </div>
+            <div class="card-description">
+              <div>
+                <h4><%= a.name %></h4>
+                <p><%= a.created_at %></p>
+              </div>
+              <p class="amount">$<%= a.amount %></p>
+            </div>
+          </div>
+          <div class="delete_wrap">
+             <%= link_to "Details",  user_category_activity_path(@user, @category, a), class: "btn" %>
+          </div>
+        </li>
+      <% end %>
+    </ul>
+  <% end %>
+</div>
+<div class="add_btn_wrap">
+  <%= link_to "Add new transaction", new_user_category_activity_path(@user, @category), class: "btn" %>
+</div>
+
+
```

Comment:
- [ ] Kindly improve the styling of the activities page to follow the design. You need to improve the card styling to match the design.

- [ ] Kindly also place the add new button at the bottom of the page and style it to follow the design

  Your Implementation | Behance Design
  :---:|:---:
  ![image](https://github.com/Kaghenimbale/budget-app/assets/36542986/eb69f9ce-f49b-4694-8714-3e225651c511) | ![image](https://github.com/Kaghenimbale/budget-app/assets/36542986/3b6cd4bd-ac62-48f9-879d-02159e37057f)